spec/patron_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'yaml'
+require './lib/patron.rb'
+
+#collection = YAML.load_file('./lib/booksdb.yml')
+patron = YAML.load_file('./lib/patrondb.yml')
```

Comment:
`patron` is not the right word for this variable. This is the patron's book list, not the patron itself.